Decred Wallet/en.lproj/Localizable.strings

```diff
@@ -410,3 +410,29 @@
 "history" = "history";
 "settings" = "Settings";
 "help" = "help";
+
+/* Backup */
+
+"keepInMind" = "Keep in mind";
+"viewSeedPhrase" = "View seed phrase";
+"reminderLabel1" = "The <bold>33-word seed phrase</bold> is EXTREMELY IMPORTANT.";
+"reminderLabel2" = "Seed phrase is <bold>the only way</bold> to restore your wallet.";
+"reminderLabel3" = "It is recommended to store your seed phrase in a <green>physical format</green> (e.g. write down on a paper).";
+"reminderLabel4" = "It is highly discouraged to store your seed phrase in any <orange>digital format</orange> (e.g. screenshot).";
+"reminderLabel5" = "Anyone with your seed phrase <orange>can steal your funds</orange>. DO NOT show it to anyone.";
+
```

Comment:
Why the empty line. i assume you are trying to group them...you can group them according to the name of each screens which should have a header